docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,329 @@ Review important information about the {kib} 8.x releases.
+* Adds numeric options list ({kibana-pull}172106[#172106]).
+Discover::
+* Hides the Empty fields section if there are no fields in it ({kibana-pull}172956[#172956]).
+* Adds tabs to Discover and Log Explorer ({kibana-pull}171467[#171467]).
+* Adds multiple improvements to saved query management ({kibana-pull}170599[#170599]).
+* Adds header row height configuration ({kibana-pull}175501[#175501]).
+* Adds new fields ingested in the background with valid mappings ({kibana-pull}172329[#172329]).
+* Adds caching to data view field list request with a `stale-while-revalidate` strategy ({kibana-pull}168910[#168910]).
+* Distinguish among empty and available fields in Discover {esql} mode ({kibana-pull}174585[#174585]).
+Elastic Security::
+For the Elastic Security 8.13.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Elastic Search::
+* Adds `ignore_empty_value` to generated set processor ({kibana-pull}175172[#175172]).
+Fleet::
+* Adds `skipRateLimitCheck` flag to the Upgrade API and Bulk_upgrade API ({kibana-pull}176923[#176923]).
+* Adds making datastream rollover lazy ({kibana-pull}176565[#176565]).
+* Stops creating  the `{type}-{datastet}@custom` component template during package installation ({kibana-pull}175469[#175469]).
+* Adds the `xpack.fleet.isAirGapped` flag ({kibana-pull}174214[#174214]).
+* Adds a warning when download upgrade is failing ({kibana-pull}173844[#173844]).
+* Adds a message explaining why an agent is not upgradeable ({kibana-pull}173253[#173253]).
+* Makes logs-* and metrics-* data views available across all spaces ({kibana-pull}172991[#172991]).
+* Adds flag for pre-release to templates/inputs endpoint ({kibana-pull}174471[#174471]).
+* Adds concurrency control to Fleet data stream API handler ({kibana-pull}174087[#174087]).
+* Adds a handlebar helper to percent encode a given string ({kibana-pull}173119[#173119]).
+Integrations::
+* Enables minute frequency for incremental syncs ({kibana-pull}176603[#176603]).
+Lens & Visualizations::
+* Adds clickable badges on compact view for {esql} queries ({kibana-pull}176568[#176568]).
+* Adds link on the documentation popover to navigate to our external docs ({kibana-pull}176377[#176377]).
+* Adds support for canceling {esql} queries in *Lens* ({kibana-pull}176277[#176277]).
+* Adds quick fixes feature to {esql} query validation errors ({kibana-pull}175553[#175553]).
+* Adds support for command settings ({kibana-pull}175114[#175114]).
+* Datatable improvements in *Lens* ({kibana-pull}174994[#174994]).
+* Adds 'greater than or equals to' and 'less than' options to filter options for date ranges and numbers ({kibana-pull}174283[#174283]).
+* Adds line breaks redesign ({kibana-pull}173596[#173596]).
+* Adds IP Prefix Aggregation-based Visualization ({kibana-pull}173474[#173474]).
+* Adds a one minute option to the date picker ({kibana-pull}172944[#172944]).
```

Comment:
This is not visualizations specific. It happens anywhere the Unified search component exists. So I would move it to a new Unified search section